backend-shared/db/clusteruser.go

```diff
@@ -197,6 +198,35 @@ func (dbq *PostgreSQLDatabaseQueries) GetClusterUserBatch(ctx context.Context, c
 		Select()
 }
 
+func (dbq *PostgreSQLDatabaseQueries) UpdateClusterUser(ctx context.Context, obj *ClusterUser) error {
+	if err := validateQueryParamsEntity(obj, dbq); err != nil {
+		return err
+	}
+
+	if err := isEmptyValues("UpdateClusterUser",
+		"clusteruser_id", obj.Clusteruser_id,
+		"user_name", obj.User_name,
+		"display_name", obj.Display_name); err != nil {
+		return err
+	}
+
+	if err := validateFieldLength(obj); err != nil {
+		return err
+	}
+
+	result, err := dbq.dbConnection.Model(obj).WherePK().Context(ctx).Update()
+	if err != nil {
+		return fmt.Errorf("error on updating clusterUser %v", err)
+	}
+
+	if result.RowsAffected() != 1 {
+		return fmt.Errorf("unexpected number of rows affected: %d", result.RowsAffected())
+	}
+
+	return nil
+
+}
+
```

Comment:
It seems like `UpdateClusterUser` isn't covered by unit tests, it couldn't hurt to add a quick test for this to improve coverage.